playbooks/openshift-node/private/clean_image.yml

```diff
@@ -27,3 +27,24 @@
     file:
       state: absent
       path: "/var/lib/cloud/"
+  - when: openshift_aws_ami_set_gquota_on_slash
+    block:
+    - name: fetch uuid of root vol
+      command: blkid /dev/xvda2 -s UUID -o value
+      register: rootvoluuid
+    - name: ensure gquota option for root vol in /etc/fstab
+      lineinfile:
+        line: "UUID={{ rootvoluuid.stdout }} /                       xfs     defaults,gquota 0 0"
+        path: /etc/fstab
+        regexp: "^UUID={{ rootvoluuid.stdout }}"
+        state: present
+    - name: set rootvol flags in grub conf
+      lineinfile:
+        line: 'GRUB_CMDLINE_LINUX="console=ttyS0,115200n8 console=tty0 net.ifnames=0 crashkernel=auto rootflags=gquota"'
+        path: /etc/default/grub
+        regexp: 'GRUB_CMDLINE_LINUX="console=ttyS0,115200n8 console=tty0 net.ifnames=0 crashkernel=auto"'
```

Comment:
How sure are we that these values will match across a wide variety of AMIs? I assume we've at least checked that they would for our cloud access AMIs?

Reply:
100% positive for the Red Hat Cloud Access AMIs. 

Ref Arch heavily stresses the use of the RHCA AMI's for EC2 based OpenShift Cluster nodes.  There is big warning about lack of support using organically developed base AMI's.